src/firestarter.cpp

```diff
@@ -176,6 +177,7 @@ void readVoltage(firestarter_handle_t* handle) {
 
     resetTimeout();
   }
+  setCommunicationMode();
```

Comment:
This line shall be inside the **if (handle->init)** block, no point to  call ```setCommunicationMode();``` every time VPP/VCC is read.